trace.go

```diff
@@ -19,13 +19,15 @@ limitations under the License.
 package trace
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
 	"path/filepath"
 	"runtime"
 	"strings"
 	"sync/atomic"
 
+	"github.com/alecthomas/template"
```

Comment:
Yeah, IIRC this is an old package and it's been fully migrated to standard `text/template`.

Reply:
I don't believe the golang templating supports the `\` which is why I pulled it in. It's not a big deal though, just need to mangle the template a bit and it's easy to eliminate.